packages/next-templates/src/app/globals.css

```diff
@@ -239,6 +239,11 @@ utrecht-alert
   padding-block: var(--voorbeeld-space-block-cat);
 }
 
+.voorbeeld-section:nth-of-type(3) {
+  background-color: var(--voorbeeld-color-white);
+  padding-block: var(--voorbeeld-space-block-cat);
```

Comment:
Misschien kan deze padding block de default zijn voor alle `.voorbeeld-section`s?

De background color is niet nodig, de parent is al wit.